content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -547,6 +547,23 @@ This annotation requires the [NodePreferAvoidPods scheduling plugin](/docs/refer
 to be enabled. The plugin is deprecated since Kubernetes 1.22.
 Use [Taints and Tolerations](/docs/concepts/scheduling-eviction/taint-and-toleration/) instead.
 
+### preemption-toleration.scheduling.sigs.k8s.io/minimum-preemptable-priority
+
+Example: `preemption-toleration.scheduling.sigs.k8s.io/minimum-preemptable-priority: "10000"`
+
+Used on: PriorityClass
+
+This annotation specifies the minimum priority value that can preempt the [PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass). It defaults to the PriorityClass's priority value +1, means pods that have a higher priority value can preempt it.
+
+### preemption-toleration.scheduling.sigs.k8s.io/toleration-seconds
+
+Example: `preemption-toleration.scheduling.sigs.k8s.io/toleration-seconds: "3600"`
+
+Used on: PriorityClass
+
+This annotation specifies how long the [PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass) can tolerate preemption by priorities lower than `MinimumPreemptablePriority`. It defaults to value 0, means the pod will be preempted immediately that is, no toleration. If
+set to a positive value, the duration will be honored. If set to a negative value, the pod can be tolerated forever that is, pods with a priority lower than `MinimumPreemptablePriority` won't be able to preempt. This affects scheduled pods only (no effect on nominated pods).
```

Comment:
You should change `MinimumPreemptablePriority` to the actual key of that annotation: `preemption-toleration.scheduling.sigs.k8s.io/minimum-preemptable-priority`. For the second use, the English phrase “minimum preemptable priority“ is fine too.

Reply:
@sftim Done